tools/ramclustr/ramclustr.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="ramclustr" name="RAMClustR" version="@TOOL_VERSION@+galaxy1">
+    <macros>
+        <import>ramclustr_macros.xml</import>
+    </macros>
```

Comment:
If you have only one tool variant, there is no need to have the parameters in the macros file. Generally, the macros file should be used for code reuse. If you plan to have multiple RAMClustR tools, ignore this comment.

You can eighter substitute the @TOOL_VERSION@ token with an actual value or you can just place the <token> tag in <macros> block inside this file. I would do the latter.

Reply:
There might be different tools or the current tool will be split into different steps, so the different tools might need different combinations of parameters.